docs/DeveloperGuide.md

```diff
@@ -1036,6 +1036,78 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes from step 2.
 
+
+**Use case: UC05 - Assign task a student**
+
+**MSS**
+1. User requests TAPA to assign a task to a student, by specifying their studentId and task name or assign tasks to all the student taking a module, by specifying module code and task name.
+2. TAPA shows the updated list of students.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. If studentId is given and a student with the given studentId does not exist.
+
+    * 1a1. TAPA shows an error message.
+
+      Use case ends.
+
+* 1b. If module code is given and no student is taking a module with the given module code.
+
+    * 1b1. TAPA shows an error message.
+
+      Use case ends.
+
+* 1c. The task name already is assigned to the present.
+
+    * 1c1. TAPA shows an error message.
+
+      Use case ends.
+
+
+**Use case: UC06 - delete task assigned to student**
+
+**MSS**
+
+1. User requests to list students.
+2. TAPA shows a list of students.
+3. User requests to delete a task that was assigned to a student by specifying their studentId and the index of the task or delete a task assigned to all students taking a certain module by specifying the module code and task name.
```

Comment:
```suggestion
3. User requests to delete a task that was assigned to a student. (This can be done for an individual student or for all the students taking the same module.)
```

Simlar to above, just a suggestion to remove some implementation details.